lib/perl/Genome/InstrumentData/AlignmentResult/Bsmap.pm

```diff
@@ -28,9 +28,10 @@ sub required_arch_os { 'x86_64' }
 # LSF resources required to run the alignment.
 sub required_rusage {
     my $self = shift;
+    my %params = @_;
     my $cores = 4; # default to 4, although decomposed_aligner_params should always include "-p N" even if the processing-profile does not
 
-    my $params = $self->decomposed_aligner_params();
+    my $params = $self->decomposed_aligner_params($params{aligner_params});
```

Comment:
If `$self` is an instance it will have the correct params already attached.  This is to handle the case where it's being called as a class method.